code/game/objects/items/weaponry.dm

```diff
@@ -960,6 +965,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
 
+/obj/item/melee/moonlight_greatsword/on_exit_storage(datum/storage/container)
+	var/obj/item/storage/belt/sheath/sword = container.real_location?.resolve()
+	if(istype(sword))
+		playsound(sword, 'sound/items/unsheath.ogg', 25, TRUE)
```

Comment:
Likewise, these should probably be an element, `/datum/element/shwing_on_shealth_exit` or something, rather than copying it across all of these items.